public/locales/en/loginDialog.json

```diff
@@ -0,0 +1,7 @@
+{
+  "welcome": "Welcome.",
```

Comment:
This shows up in the dashboard too. Can we just place it in common and have both components read from it? I think common is a good place to place individual words we might place all around the page.